core/workspace_svg.js

```diff
@@ -725,6 +733,17 @@ Blockly.WorkspaceSvg.prototype.getSvgXY = function(element) {
   return new Blockly.utils.Coordinate(x, y);
 };
 
+/**
+ * Gets the size of the workspace's parent svg element.
+ * @return {!Blockly.utils.Size} The cached width and height of the workspace's
+ *     parent svg element.
+ * @package
+ */
+Blockly.WorkspaceSvg.prototype.getCachedParentSvgSize = function() {
+  var size = this.cachedParentSvgSize_;
```

Comment:
Copying to avoid handing out the actual object, right?

Reply:
Yup! This was causing a super fun bug this morning : )